src/router/index.js

```diff
@@ -33,15 +32,6 @@ Vue.use(VueRouter)
 const routes = [
     {
         path: '/',
-        redirect: { name: 'LoadingView' },
-    },
-    {
-        path: '/startup',
-        name: 'LoadingView',
-        component: LoadingView,
-    },
-    {
-        path: '/map',
```

Comment:
Have you looked at the routing work that I've done in #130 ? This will conflict with this commit. Why removing the `LoadingView` ? It might be a good idea to discuss the goal of this LoadingView and some edge case like for instance if mf-chsdi3 is not available.